foundation/foundation-mda/src/main/resources/templates/data-delivery-data-records/data-delivery-separate-data-records-java.pom.xml.vm

```diff
@@ -30,6 +30,7 @@
         <dependency>
             <groupId>org.codehaus.jackson</groupId>
             <artifactId>jackson-mapper-asl</artifactId>
+            <version>${version.jackson.mapper.asl}</version>
```

Comment:
Q: Will there be a baton migration for this?

Reply:
Yeah, we should but given the time of the ticket. This migration will be added in a follow up ticket.

Reply:
Discussed offline. Since this is a simple migration step and given the split data record not popularly used, most importantly, without adding this version tag, the build can't be run, we will just add this as breaking changes to the release note.